modules/@ergonode/products/src/extends/components/Tabs/ProductVariantsTab.vue

```diff
@@ -306,13 +313,26 @@ export default {
                 filters,
                 sortedColumn,
             };
+            const filteredProductTypes = await this.$extendMethods('@Products/components/Tabs/ProductVariantsTab/filteredProductTypes', {
+                $this: this,
+            });
+            const productTypes = [
+                ...new Set([].concat(...filteredProductTypes)),
```

Comment:
Array.from(new Set(filteredProductTypes)) - it's going to be much more simple and faster

Reply:
`filteredProductTypes` is a two-dimensional array. I must contact arrays and remove duplications.